src/sass/layouts/_footer.scss

```diff
@@ -1,2 +1,30 @@
 @use '../utilities/variables';
 
+.footer__container {
+  height: 100px;
+  width: 100%;
+  background-color: #f7f7f7; // change to variable
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  bottom: 0;
+  .footer__container-text {
+    width: 320px;
```

Comment:
**footer contaner-text width: 320px**
To achieve the responsive design this will need to be a min-width of 230pxish, along with the paddings i spoke of above. You will have to find the exact number that gets your line break where the layout has. Ive found the actual width to be a bit less that what the layout shows.